src/telemetry/telemetryService.ts

```diff
@@ -304,7 +305,9 @@ export default class TelemetryService {
 
         this.track(TelemetryEventTypes.NEW_CONNECTION, preparedProperties);
       }
-    });
+    } catch (error) {
+      log.error('TELEMETRY track new connection', error);
```

Comment:
Would it make sense to track the telemetry error here? (We'd probably want to sanitize the error somehow though)

Reply:
Max hasn't requested a telemetry event for failed connections, only for ones that succeeded.

Reply:
Gotcha - I was thinking about creating a telemetry event around letting us know we've missed tracking a connection because we errored parsing the connection instance information, not an event around a connection failing to connect. It's probably something we'd want a bug tracker to report, but since we don't have one it might make sense to report it to telemetry so we're aware if users are adding connections but we aren't properly tracking them.

Reply:
Good idea, pls create a ticket to triage it!